app/models/user.rb

```diff
@@ -35,6 +35,15 @@ def early_career_teacher?
     early_career_teacher_profile.present?
   end
 
+  def mentor?
+    mentor_profile.present?
+  end
+
+  def core_induction_programme
+    return early_career_teacher_profile.core_induction_programme if early_career_teacher?
+    return mentor_profile.core_induction_programme if mentor?
+  end
```

Comment:
I don't think this should live on user class. User on its own is just an abstract persona using our service - it should not know about core_induction_programme. I''m relatively happy to add this for now, but I guess we should be striving to reduce use of this class for everything as we are heading for afferent/efferent combo here